userspace/libsinsp/cri.h

```diff
@@ -27,33 +27,133 @@ limitations under the License.
 
 #include "container_info.h"
 
+#ifdef GRPC_INCLUDE_IS_GRPCPP
+#	include <grpcpp/grpcpp.h>
+#else
+#	include <grpc++/grpc++.h>
+#endif
+
 namespace libsinsp {
 namespace cri {
+
+// these shouldn't be globals but we still need references to *the* CRI runtime
```

Comment:
Agree. Do you want to open a ticket to track the removal of globals in the future?

Reply:
I opened https://github.com/draios/sysdig/issues/1536